app/assets/stylesheets/mission_control/jobs/jobs.css

```diff
@@ -5,3 +5,8 @@
 .jobs td {
     word-break: break-all;
 }
+
+.job .arguments {
+    max-height: 20rem;
```

Comment:
Maybe even 10rem is fine:

<img src="https://github.com/rails/mission_control-jobs/assets/47774/d96d7ec9-ba60-4ced-901d-932d6ff209b9" width="336px" />